.github/workflows/byob-slsa.yaml

```diff
@@ -0,0 +1,28 @@
+# This builds a SLSA provenance statement based on BYOB.
+# For now it is under heavy development and is not yes suited for releases.
+---
+name: SLSA Provenance
+on:
+  - workflow_dispatch
+
+permissions: read-all
+
+env:
+  GH_TOKEN: ${{ github.token }}
+  ISSUE_REPOSITORY: ${{ github.repository }}
+jobs:
+  usetrw:
+    permissions:
+      contents: write
+      id-token: write
+      actions: read
+      packages: write
+    uses: AdamKorcz/java-slsa-generator/.github/workflows/gradle-trw.yml@main
+    with:
+      rekor-log-public: true
+      artifact-list: |
+        ./sigstore-java/build/local-maven-repo/dev/sigstore/sigstore-java/GRADLE_VERSION/sigstore-java-GRADLE_VERSION.module,
```

Comment:
the documentation says this should be GRADLEVERSION? https://github.com/AdamKorcz/java-slsa-generator/blob/main/.github/workflows/gradle-trw.yml#L28

Reply:
Was a mistake in the documentation.